deploy/chart/devfile-registry/values.yaml

```diff
@@ -20,13 +20,22 @@ global:
     secretName: ''
   isOpenShift: false
   tlsEnabled: false
+  headless: false
 
 devfileIndex:
   image: quay.io/devfile/devfile-index
   tag: next
   imagePullPolicy: Always
   memoryLimit: 256Mi
 
+registryViewer:
+  image: quay.io/devfile/registry-viewer
+  tag: next
+  imagePullPolicy: Always
+  memoryLimit: 256Mi
+  registryHosts: |
+    "[{ \"name\": \"Community\", \"link\": \"http://localhost:8080\" }]"
```

Comment:
TBH, I don't think we need to store this in the values.yaml, since I don't think users will be configuring it. We could keep the current entry that was removed from the configmap, and mount it as an environment variable in the viewer container (see my other comment relating to this)